wpilibNewCommands/src/test/java/edu/wpi/first/wpilibj2/command/CommandDecoratorTest.java

```diff
@@ -182,4 +182,27 @@ void perpetuallyTest() {
       assertTrue(scheduler.isScheduled(perpetual));
     }
   }
+
+  @Test
+  void unlessTest() {
```

Comment:
maybe split this test into two? I'm a bit concerned about leftover state.

using parameterized tests, you can just have the same test, just once for `true` and once for `false`.
not sure how you'd do this in C++ though

Reply:
In my mind at least, that is kinda the point of the test. The command didn't run when the value was false but the same command ran once the value changed and it was scheduled again.